src/inmanta/data/model.py

```diff
@@ -36,45 +36,15 @@
 import pydantic_core.core_schema
 from inmanta import const, data, protocol, resources
 from inmanta.stable_api import stable_api
-from inmanta.types import ArgumentTypes, JsonType
+from inmanta.types import ArgumentTypes
+from inmanta.types import BaseModel as BaseModel  # Keep in place for backwards compat with <=ISO8
+from inmanta.types import JsonType
 from inmanta.types import ResourceIdStr as ResourceIdStr  # Keep in place for backwards compat with <=ISO8
 from inmanta.types import ResourceType as ResourceType  # Keep in place for backwards compat with <=ISO8
 from inmanta.types import ResourceVersionIdStr as ResourceVersionIdStr  # Keep in place for backwards compat with <=ISO8
 from inmanta.types import SimpleTypes
 
 
-def api_boundary_datetime_normalizer(value: datetime.datetime) -> datetime.datetime:
```

Comment:
Moved this whole block to `inmanta.types` because it's used in type aliases there. It always was, and I suspect it always was a bit of a problem in some way, but my changes to the type aliases finally pushed it over the brink and crashed the swagger generator.